rest-server/templates/src/main/scala/com/actionml/templates/cb/CBEngine.scala

```diff
@@ -68,7 +75,8 @@ class CBEngine() extends Engine() with JsonParser {
   }
 
   /** Triggers parse, validation, and persistence of event encoded in the json */
-  def input(json: String, trainNow: Boolean = true): Validated[ValidateError, Boolean] = {
+  // Slava this needs to stay "input" why did you change the API?
+  def inputInternal(json: String, trainNow: Boolean = true): Validated[ValidateError, Boolean] = {
```

Comment:
I merged your personal develop branch. @snovgorodtsev, please push correction to `actionml/harness/mirror`